style.css

```diff
@@ -38,29 +48,37 @@ html {
 
 /* HEADLINE  */
 .headline {
-  display: flex;
-  flex-direction: column;
-  gap: 2.4rem;
   background: url(images/header-background.png);
   background-repeat: no-repeat;
   background-size: cover;
   padding: 2.8rem;
+  background-position: 100% auto;
```

Comment:
- [ ] While working on the desktop version, we should also make sure that the mobile version layout is maintained. Kindly make sure the background image looks the same as the design. 🤓️


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/236136203-24ef90cd-1379-4160-9450-92223bf153a2.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/236136218-2c0d6100-d2e1-4732-af41-4e1b19a57e4c.png" width="300">|